cmd/diff.go

```diff
@@ -51,13 +51,13 @@ func init() {
 		false, "do not diff consumers or "+
 			"any plugins associated with consumers")
 	diffCmd.Flags().IntVar(&diffCmdParallelism, "parallelism",
-		10, "Maximum number of concurrent operations")
+		10, "Maximum number of concurrent operations.")
 	diffCmd.Flags().StringSliceVar(&dumpConfig.SelectorTags,
 		"select-tag", []string{},
 		"only entities matching tags specified via this flag are diffed.\n"+
-			"Multiple tags are ANDed together.")
+			"When there are multiple tags, entities must match all tags.")
```

Comment:
```suggestion
			"When this setting has multiple tag values, entities must match each of them.")
```

"there are multiple tags" makes me raise the question "where are they?". This is my attempt at clarifying that.